sdk/ai/ai-projects/src/agents/utils.ts

```diff
@@ -28,3 +28,31 @@ export function fromFunctionDefinition(functionDefintion: FunctionDefinition): F
         function: functionDefintion
     }
 }
+
+/** Types of connection tools sued to configure an agent */
+export enum connectionToolType {
+  /** Bing grounding search tool */
+  BingGrounding = "bing_grounding",
+  /** Microsoft Fabric tool */
+  MicrosoftFabric = "microsoft_fabric",
+  /** Sharepoint tool */
+  SharePointGrounding = "sharepoint_grounding",
+  /** Azure AI search tool */
+  AzureAISearch = "azure_ai_search",
+}
+
+/**
+ * Creates a tool definition for a connection tool with the given connection ids.
+ *
+ * @param toolType - The type of the connection tool.
+ * @param connectionIds - A list of the IDs of the connections to use.
+ * @returns The function tool definition.
+ */
+export function fromConnectionIds(toolType : connectionToolType, connectionIds: string[]) : ToolDefinitionParent {
```

Comment:
Are there other properties needed for each tool definition apart from connections? If so, how would those initialized?

Reply:
All the tools need are the connection IDs